cmd/osv-scanner/fixtures/osv-scanner-complex-licenses-config.toml

```diff
@@ -0,0 +1,23 @@
+[[PackageOverrides]]
+ecosystem = "npm"
+ignore = true
+license.override = ["0BSD"]
+
+[[PackageOverrides]]
+ecosystem = "Packagist"
+license.override = ["0BSD"]
+
+[[PackageOverrides]]
+ecosystem = "Alpine"
+name = "apk-tools"
+license.ignore = true
+license.override = ["UNKNOWN"] # this takes priority over license.ignore
```

Comment:
note: I realised while doing this that the new composition logic combines with the order to allow you to selectively override specific packages under a particular condition while having related packages have different logic, though only if the logic is truthy

For now I think it's worth keeping as an undocumented feature given you cannot do e.g. `ignore = false` (as that's a negative condition), but it's cool and might be worth exploring further in future

Reply:
I do think `license.ignore` should take the priority - is this hard to implement?

Reply:
done